README.md

```diff
@@ -2,7 +2,7 @@
 
 # Winnow
 
-An AREL based search solution for Rails.
+An [AREL](https://github.com/rails/arel/tree/6-0-stable) based search solution for Rails.
```

Comment:
AREL master got moved into Active Record, but this gem is using the v6.something so linking to that version. Maybe that's a less good idea? Not sure.

Reply:
Is there a way we can link to docs that won't change often? I don't like the idea of having to update links.

Reply:
Oh, I linked to branch, not the tag. Oops.
Would this be better?
https://github.com/rails/arel/tree/v6.0.4
I was thinking of replacing (in Admin at least) the whole thing with ransack https://github.com/activerecord-hackery/ransack
if only because it allows for filter/search across associations.. but that's a larger discussion.

Reply:
I think https://github.com/rails/arel/tree/v6.0.4 is better. Looks like Arel is no longer a seperate gem once it got bundled with ActiveRecord.

I think that ransack is worth looking at - but need to consider the implications for the full text search fields.

Reply:
```suggestion
An [AREL](https://github.com/rails/arel/tree/v6.0.4) based search solution for Rails.
```